<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проверка на антиплагиат</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QGroupBox" name="settingsGroup">
      <property name="title">
       <string>Настройки проверки</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Порог срабатывания (%):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="thresholdSpinBox">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>70.000000000000000</double>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkButton">
         <property name="text">
          <string>Проверить на плагиат</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold; padding: 8px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QGroupBox" name="text1Group">
        <property name="title">
         <string>Исходный текст</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QTextEdit" name="originalTextEdit">
           <property name="placeholderText">
            <string>Введите исходный текст здесь...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text1StatsLabel">
           <property name="text">
            <string>Символов: 0, Слов: 0</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #7f8c8d; font-size: 11px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="text2Group">
        <property name="title">
         <string>Проверяемый текст</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QTextEdit" name="checkedTextEdit">
           <property name="placeholderText">
            <string>Введите текст для проверки здесь...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text2StatsLabel">
           <property name="text">
            <string>Символов: 0, Слов: 0</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #7f8c8d; font-size: 11px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="resultGroup">
      <property name="title">
       <string>Результат проверки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="similarityLabel">
         <property name="text">
          <string>Схожесть: 0.00%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px; font-weight: bold; color: #27ae60;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="detailsTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Здесь будет отображаться подробная информация о проверке...</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family: 'Courier New'; font-size: 11px; background-color: #f8f9fa;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>